meson.build

```diff
@@ -278,7 +281,10 @@ if get_option('fish-completions')
 		'completions/fish/swaynag.fish',
 	)
 	if fish_comp.found()
-		fish_install_dir = fish_comp.get_pkgconfig_variable('completionsdir')
+		fish_install_dir = fish_comp.get_pkgconfig_variable(
+			'completionsdir',
+			define_variable: ['datadir', datadir]
```

Comment:
`datadir` is relative to the `prefix`. We need to use the full path here: `prefix / datadir`.

Reply:
bah stupid github doesn't send comment when you resolve... retyping -_-

`meson configure` does display a relative path, but it sends the full path here -- I tested it, fish completion is properly installed within the prefix.

Reply:
Ah, that's likely because the resulting string is fed into `install_dir`, which accepts paths relative to `prefix`. Should be fine then!